static/js/notes.js

```diff
@@ -0,0 +1,55 @@
+var USERS_BY_ID = {};
+
+const $createAuthor = function(authorId) {
```

Comment:
Nit: these createX should probably be more like createXDom or createXHtml.

Reply:
Yeah I can see how that can be confusing. I made a rule with myself that if a function returns a jQuery object it starts with a `$` which makes saying DOM or HTML at the end of the function somewhat redundant

Reply:
not pure html, but a jQuery object

Reply:
I would probably call that `createAuthorElement` or if I knew more specifically what it was I might say that. For example `createAuthorDiv` or `createAuthorListItem`.